src/chat/chatSelectors.js

```diff
@@ -39,3 +40,20 @@ export const getRecentConversations = (state) => {
     .map(recipient => recipient[0]);
   return sortedByTimestamp;
 };
+
+export const getCurrentTypingUser = state => {
```

Comment:
What does this function do? What if there are multiple users typing in a narrow

Also please add comments